CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,159 @@
+# pyOpenSci Community Code of Conduct
+
+pyOpensci is dedicated to creating a welcoming, supportive and diverse 
+community around the open source Python tools that drive open
+science. We do this through a combination of community building, 
+mentorship, peer review of software and education and outreach. 
+
+We are building the pyOpenSci community in a way that promotes fairness 
+and justice for all particular focus on supporting those from 
+traditionally marginalized groups. We want every member of our community 
+to thrive; we want to make a positive impact on our community members work 
+and on the work of others. 
+
+### What this code of conduct defines 
+
+Our Code of Conduct defines expected behavior 
+and guidelines that help create such a community.
+
+Anyone who participates in a pyOpenSci activity either online or in person is expected to show respect and courtesy to others in all interactions. These interactions may occur in a variety of locations including:
+
+* in GitHub repositories
+* in our Slack channel
+* in our Discourse online forum 
+* during in-person events, 
+* when representing pyOpenSci in public, 
+* or in any event online or in person where pyOpenSci is involved.  
+
+To make sure that everyone has a common understanding of “show respect and courtesy to each other,” we have adopted the following code of conduct. The code of conduct is enforced by the pyOpenSci Executive Director and the Advisory Council.
+
+## Unacceptable behavior
+
+The following types of behavior are unacceptable in pyOpenSci spaces, both online and in-person, and constitute code of conduct violations.
+
+### Abusive behavior
+
+**Harassment:** including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion, as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
+
+**Threats:** threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.
+
+### Unwelcoming behavior
+
+#### Blatant acts and micro-aggressions
+Blatant micro-aggressions involve making statements that assume 
+negative characteristics in a blanket fashion because of someone's identification 
+with a particular group. We take micro-aggressions seriously,  especially 
+important when they target traditionally marginalized groups (e.g., 
+explicitly racist, sexist, homophobic, or transphobic statements).
+
+ > For example, arguing that some people are less intelligent because 
+ > of their gender, race, religion. 
+ 
+#### Subtle and unintentional micro-aggressions 
```

Comment:
Repeated, subtle, and unintentional micro-agressions? 

I think it is worth trying to clearly signal in this section that there is obviously bad behaviour

Reply:
thank you! updated the heading there!